deploy/build

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Run the docs build process twice, producing `build/latest` and
+# `build/stable` instead of `docs/_build`.
+
+set -e
+
+
+latest_branch=main
+stable_tag=$(git tag --sort=version:refname | tail -1)
```

Comment:
In my testing, this works well to identify the correct version and respects post-release suffixes like `-1` and -2`. It will treat a tag like `-rc1` as being _newer_ than the release it refers to, but we currently don't use those in `securedrop-docs`.

Reply:
If we do end up needing to handle `-rc` versions, it looks like we can configure Git for that: https://git-scm.com/docs/git-config#Documentation/git-config.txt-versionsortsuffix